docs/usage.md

```diff
@@ -1,46 +1,45 @@
 # Basic Usage
 
-`jrnl` has two modes: **composing** and **viewing**. Basically, whenever
-you _don't_ supply any arguments that start
-with a dash or double-dash, you're in composing mode, meaning you can
-write your entry on the command line or an editor of your choice.
-
-We intentionally break a convention on command line arguments: all
-arguments starting with a _single dash_
-will _filter_ your journal before viewing
-it, and can be combined arbitrarily. Arguments with a
-_double dash_ will control how your journal
-is displayed or exported and are mutually exclusive (ie. you can only
-specify one way to display or export your journal at a time).
+`jrnl` has two modes: **composing** and **viewing**. Whenever you don't enter
+any arguments that start with a dash (`-`) or double-dash (`--`), you're in
+composing mode, meaning that you can write your entry on the command line.
+
+We intentionally break a convention on command line arguments: all arguments
+starting with a _single dash_ (`-`) will _filter_ your journal before viewing
+it. Filter arguments can be combined arbitrarily. Arguments with a _double dash_
+(`--`) will _control_ how your journal is displayed or exported. Control
+arguments are mutually exclusive (i.e., you can only specify one way to display
+or export your journal at a time).
 
 ## Listing Journals
 
-You can list the journals accessible by jrnl
+You can list the journals accessible by `jrnl`:
 
 ```sh
 jrnl -ls
 ```
 
-The journals displayed correspond to those specified in the jrnl
-configuration file.
+The journals displayed correspond to those specified in the `jrnl` configuration
+file.
 
 ## Composing Entries
 
-Composing mode is entered by either starting `jrnl` without any
-arguments -- which will prompt you to write an entry or launch your
-editor -- or by just writing an entry on the prompt, such as
+Composing mode is entered by either starting `jrnl` without any arguments --
+which will prompt you to write an entry or launch your editor -- or by just
+writing an entry on the prompt, such as
 
 ```sh
-jrnl today at 3am: I just met Steve Buscemi in a bar! He looked funny.
+jrnl today at 3am: I just met Steve Buscemi in a bar! He was on fire.
```

Comment:
Good catch! This is definitely against our code of conduct. Thank you for holding our feet to the fire. 🔥

Reply:
lol, pun 🤣